docs/IR.md

```diff
@@ -1,78 +1,210 @@
-Open Neural Network Exchange (ONNX)
+Open Neural Network Exchange (ONNX) v1
 =========
 
-ONNX is an open specification that consists of the following components:
-
-1)  A definition of an extensible computation graph model
+__Purpose__
 
-2)  Definitions of built-in operators and standard data types
+This document contains the normative specification of the semantics of ONNX. The .proto and .proto3 files found under the ‘onnx’ folder form the normative specification of its syntax. Commentary found in the .proto and .proto3 files are intended to improve readability of those files, but are not normative if they conflict with this document. Such conflicts should be reported as documentation bugs.
```

Comment:
shall we call out that the model checker script validates for all these rules?

Reply:
Does it? How?

Reply:
it should validate against the rules you describe in the document (uniqueness, format, etc). if it's in the spec but not in the checker, I worry we will slowly deviate from the spec. 
I'm sure the current checker does not validate all the rules currently, in which case there are some work items to remediate.

Reply:
Oh, I see. I thought you were saying that it was validating that the semantic spec and the comments in the proto files were the same, and that didn't make any sense.

I'll add a comment describing what you said.